task_hive/pubspec.yaml

```
@@ -46,6 +46,10 @@ dependencies:
   supabase_flutter: ^2.8.4
   flutter_dotenv: ^5.2.1
   rxdart: ^0.28.0
+#  file_picker: ^5.3.2
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)